app/javascript/crayons/Button/DangerButton.jsx

```diff
@@ -0,0 +1,32 @@
+import { h } from 'preact';
```

Comment:
Ok, I'm sure you thought about it, but do we need these specialized files? They seem identical except for the name of the component. They all use `<Button variant="VARIANT_NAME">`, what if we just use `Button` ? And if and when we need one with lots of custom behavior, that can be added.

What do you think?

Reply:
I decided to go with the separate buttons mainly because I thought from a DX perspective, `<DangerButton />` would be more grokkable than `<Button variant="danger" />`, but this is not a deal breaker for me.

Reply:
+1 on having less code in general :-) Also ripgrep is quite fast so `rg Button | rg danger` is not a big issue :D

Reply:
Yeah. I asked some others on our Slack, and I think I will just go with `<Button variant="..." />` without the additional layer.

Reply:
Now there is only the `<Button />` component and you simply use the variant `prop` to decide what type of button you want. e.g. `<Button variant="danger">DANGER!</Button>`